### PR TITLE
🐛   Set error_exit code arg in kubeadm bootstrap script

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: The error_exit func expects a message (`$1`) and code (`$2`) arg, but some places in the script set a message arg, resulting in this error when an error occurs:

```
[  118.164962] cloud-init[1769]: [2021-01-14 19:55:48] !!! [2021-01-14T19:55:48+00:00] cluster.x-k8s.io kubeadm bootstrap script /usr/local/bin/kubeadm-bootstrap-script exiting with status
[  118.173866] cloud-init[1769]: [2021-01-14 19:55:48] /usr/local/bin/kubeadm-bootstrap-script: line 29: exit: : numeric argument required
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh
Comment (CecileRobertMichon) [inline bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh:105]: not sure if we want to set a different exit code or pass through the kubeadm_return

/cc @randomvariable

**Comment by CecileRobertMichon** [comment]
/hold

need to run make generate

**Comment by fabriziopandini** [comment]
lgtm for me, but I leave the final word to @randomvariable

**Review by vincepri — APPROVED** [review]
LGTM

/assign @randomvariable 
/milestone v0.4.0

Should we backport this?

File: bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh
Comment (randomvariable) [reply bootstrap/kubeadm/internal/cloudinit/kubeadm-bootstrap-script.sh:105]: I think pass through is fine.

**Comment by randomvariable** [comment]
Thanks for this. I had seen that line crop up before, but didn't know what it was.

/lgtm

**Pull request merged**
